tests/test_inspect.py

```diff
@@ -0,0 +1,154 @@
+    pytest.importorskip("inspect_ai")
+
+    turn = Turn("user", "Hello!")
+    messages = turn_as_messages(turn)
+
+    assert len(messages) == 1
+    assert isinstance(messages[0], ChatMessageUser)
+
+
+def test_turn_as_messages_assistant():
+    pytest.importorskip("inspect_ai")
+
+    turn = Turn("assistant", "Hi there!")
+    messages = turn_as_messages(turn, model="gpt-4")
+
+    assert len(messages) == 1
+    assert isinstance(messages[0], ChatMessageAssistant)
+    assert messages[0].model == "gpt-4"
+
+
+def test_content_to_inspect_text():
+    pytest.importorskip("inspect_ai")
+
+    content = ContentText(text="Hello world")
+    inspect_content = content_to_inspect(content)
+
+    assert isinstance(inspect_content, IContentText)
+    assert inspect_content.text == "Hello world"
+
+
+def test_content_to_inspect_image_remote():
+    pytest.importorskip("inspect_ai")
+
+    content = ContentImageRemote(url="https://example.com/image.jpg", detail="high")
+    inspect_content = content_to_inspect(content)
+
+    assert isinstance(inspect_content, ContentImage)
+    assert inspect_content.image == "https://example.com/image.jpg"
+    assert inspect_content.detail == "high"
+
+
+def test_content_to_inspect_json():
+    pytest.importorskip("inspect_ai")
+
+    content = ContentJson(value={"key": "value", "number": 42})
+    inspect_content = content_to_inspect(content)
+
+    assert isinstance(inspect_content, ContentData)
+    assert inspect_content.data == {"key": "value", "number": 42}
+
+
+def test_content_to_chatlas_string():
+    pytest.importorskip("inspect_ai")
+
+    result = content_to_chatlas("Hello world")
+
+    assert isinstance(result, ContentText)
+    assert result.text == "Hello world"
+
+
+def test_content_to_chatlas_text():
+    pytest.importorskip("inspect_ai")
+
+    inspect_content = IContentText(text="Test message")
+    result = content_to_chatlas(inspect_content)
+
+    assert isinstance(result, ContentText)
+    assert result.text == "Test message"
+
+
+def test_content_to_chatlas_image_url():
+    pytest.importorskip("inspect_ai")
+
+    inspect_content = ContentImage(image="https://example.com/test.jpg", detail="low")
+    result = content_to_chatlas(inspect_content)
+
+    assert isinstance(result, ContentImageRemote)
+    assert result.url == "https://example.com/test.jpg"
+    assert result.detail == "low"
+
+
+def test_content_to_chatlas_json():
+    pytest.importorskip("inspect_ai")
+
+    inspect_content = ContentData(data={"test": "data", "value": 123})
+    result = content_to_chatlas(inspect_content)
+
+    assert isinstance(result, ContentJson)
+    assert result.value == {"test": "data", "value": 123}
+
+
+def test_content_to_inspect_tool_request_raises():
+    pytest.importorskip("inspect_ai")
+
+    content = ContentToolRequest(id="call_123", name="test_tool", arguments={})
+
+    with pytest.raises(ValueError, match="cannot be directly translated"):
+        content_to_inspect(content)
```

Comment:
A better test would be to check whether actual `Turn`s with tool contents get translated correctly to inspect messages (i.e., `turn_as_messages()`)